satellites/main.yaml

```diff
@@ -0,0 +1,8 @@
+
+sources:
+
+  sat:
+    description: Satellite data and products from different sources
```

Comment:
There have been satellite data products mixed into this PR. They should be handled separately.

Reply:
Sry 🙈 I'll change the pull request.